package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "argo-clinical",
-  "version": "1.77.0",
+  "version": "1.78.0",
```

Comment:
Don't need to bump version when merging into dev. Also, this size of fix should be a patch not minor version bump.